grails-app/assets/javascripts/streama/templates/modal--manage-files.tpl.htm

```diff
@@ -131,26 +129,29 @@ <h4>Subtitles</h4>
         <tr>
           <th>ID</th>
           <th>Name</th>
-          <th>Type</th>
-          <th>Label</th>
           <th>Language</th>
-          <th style="width: 68px;"></th>
+          <th>Label</th>
+          <th style="width: 235px; text-align: center">Actions</th>
         </tr>
         </thead>
         <tbody>
-        <tr ng-repeat="file in video.subtitles">
+        <tr ng-repeat="file in video.subtitles | orderBy: 'id'">
           <td>{{file.id}}</td>
-          <td>{{file.originalFilename}}</td>
-          <td>{{file.contentType}}</td>
-          <th>
-            <input class="form-control input-xs" value="{{file.subtitleLabel}}" placeholder="Country" ng-change="subtitleLabel" ng-model="file.subtitleLabel"></input>
-          </th>
-          <th>
-            <input class="form-control input-xs" value="{{file.subtitleSrcLang}}" placeholder="ISO-3166 Country Code" ng-change="subtitleSrcLang" ng-model="file.subtitleSrcLang"></input>
-          </th>
-          <td style="padding: 6px;">
-            <button title="Delete file" class="btn btn-xs btn-danger" ng-click="removeFile(file)"><i class="ion-trash-a"></i></button>
-            <button title="" class="btn btn-xs btn-primary" ng-click="saveChanges(file)" ><i class="ion-checkmark"></i></button>
+          <td style="overflow:hidden; text-overflow:ellipsis; max-width: 375px; white-space: nowrap;">{{file.originalFilename}}</td>
```

Comment:
This might be better served as a CSS class rather than inline style, same for the occurences below